src/main/java/org/apache/rocketmq/dashboard/controller/DlqMessageController.java

```diff
@@ -83,7 +83,7 @@ public Object batchResendDlqMessage(@RequestBody List<DlqMessageRequest> dlqMess
         return dlqMessageService.batchResendDlqMessage(dlqMessages);
     }
 
-    @PostMapping(value = "/batchExportDlqMessage.do")
```

Comment:
The query suffix is not appropriate for interfaces of the operation type.

Reply:
exportDlqMessage.query, there are 2 verbs in this interface :-)

Reply:
> The query suffix is not appropriate for interfaces of the operation type.

Offline synced, there will be another PR to refactor.